src/scaffold.test.ts

```diff
@@ -1,24 +1,50 @@
 import * as assert from "assert";
-import * as scaffold from "./scaffold";
+import sinon from "sinon";
+import * as vscode from "vscode";
+import { pickTemplate } from "./projectGeneration/templates";
 
-describe("filterSensitiveKeys", () => {
-  it("filters out keys containing 'key' or 'secret'", () => {
-    const input = {
-      api_key: "sensitive",
-      secret_token: "sensitive",
-      bootstrap_server: "localhost:9092",
-      topic_name: "test-topic",
-    };
+describe("template sorting", () => {
+  let sandbox: sinon.SinonSandbox;
 
-    const result = scaffold.filterSensitiveKeys(input);
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("should present templates to the user sorted by display_name", async () => {
+    const unsortedTemplates = [
```

Comment:
This ... looks sorted already!

Reply:
ok, adjusted the one in `scaffold` to be a real test. removed the `templates` one because it ended up looking super similar and became redundant